docs/adopters.md

```diff
@@ -52,8 +54,9 @@ The following are a list of TiKV adopters deploying TiKV with TiDB.
 |[Hoodinn](https://www.crunchbase.com/organization/hoodinn)|Gaming||
 |[SEA group](https://sea-group.org/?lang=en)|Gaming||
 |[Zhaopin.com](https://www.crunchbase.com/organization/zhaopin)|Recruiting||
+|[BIGO](https://www.crunchbase.com/organization/bigo-technology); |Live Streaming|[English](https://en.pingcap.com/case-studies/why-we-chose-an-htap-database-over-mysql-for-horizontal-scaling-and-complex-queries);[Chinese](https://pingcap.com/cases-cn/user-case-bigo/)|
```

Comment:
```suggestion
|[BIGO](https://www.crunchbase.com/organization/bigo-technology) |Live Streaming|[English](https://en.pingcap.com/case-studies/why-we-chose-an-htap-database-over-mysql-for-horizontal-scaling-and-complex-queries); [Chinese](https://pingcap.com/cases-cn/user-case-bigo/)|
```

Reply:
![image](https://user-images.githubusercontent.com/14819777/95968036-932f2100-0e3f-11eb-9d17-02954674be8e.png)
Is `BiGO;` intended?

Reply:
removed, and thanks!